_data/simple-icons.json

```diff
@@ -6595,6 +6595,12 @@
             "hex": "4285F4",
             "source": "https://commons.wikimedia.org/wiki/File:Google_Translate_logo.svg"
         },
+        {
+            "title": "Google TV",
+            "hex": "4285F4",
+            "source": "https://tv.google",
```

Comment:
```suggestion
            "source": "https://partnermarketinghub.withgoogle.com/brands/google-tv/visual-identity/visual-identity/",
```

Reply:
The contributers.md said I should include a link where the logo is being used, hence the original website should be valid right? I had also seen other Google Products in the codebase being linked to the original website. I had included the actual guidelines below that line with that link you suggested. Let me know what you think, and I can work on it, thanks :)

Reply:
Sure, but better assets are linked directly on that brand page. We should prefer the assets that they explicitly provide for us to use.

Reply:
Awesome sauce! I will accept those changes